test/test_van/test_email.py

```diff
@@ -81,6 +122,15 @@ def assert_matching_tables(table1, table2, ignore_headers=False):
     },
 ]
 
+mock_response_enriched = deepcopy(mock_response)
```

Comment:
This enriched table includes the output you would get with `expand=True`. Last entry gets the A/B test.

Reply:
In our pair programming session today, @matthewkrausse pointed out that this sample/fake data may not correspond to the actual data. Do you have actual data to compare it too? If not, we can figure something out.